<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>430</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>  Ideal Weight</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Favicon.qrc">
    <normaloff>:/Favicon/фавикон.png</normaloff>:/Favicon/фавикон.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#MainWindow{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0.875, stop:0 #f06d5b, stop:1 rgb(255, 35, 112));
}
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>71</width>
      <height>77</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Simplex_IV25</family>
      <pointsize>17</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
/*font-family: &quot;Montserrat&quot;,Verdana,sans-serif;*/
font: 17pt /*&quot;Segoe UI&quot;*/&quot;Simplex_IV25&quot;;
font-weight: 600;</string>
    </property>
    <property name="text">
     <string>ПОЛ</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>360</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider#horizontalSlider::handle {
	background-color: rgb(255, 255, 255);
	width: 10px;
	height: 10px;
    border-radius: 5px;

 }
 /*QSlider#horizontalSlider::sub-page:horizontal {
     background: rgb(49, 255, 46);
 }*/</string>
    </property>
    <property name="minimum">
     <number>120</number>
    </property>
    <property name="maximum">
     <number>220</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>310</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Simplex_IV25</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
/*font-family: &quot;Montserrat&quot;,Verdana,sans-serif;*/
font: 15pt /*&quot;Segoe UI&quot;*/&quot;Simplex_IV25&quot;;
font-weight: 600;</string>
    </property>
    <property name="text">
     <string>РОСТ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
/*font-family: &quot;Montserrat&quot;,Verdana,sans-serif;*/
font: 15pt &quot;Segoe UI&quot;;
font-weight: 500;</string>
    </property>
    <property name="text">
     <string>ЕДИНИЦА ИЗМЕРЕНИЯ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Simplex_IV25</family>
      <pointsize>17</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 157, 148);
	color: rgb(255, 255, 255);
	border: 1px solid gray;
	border-radius: 20px;
	font-family: /*&quot;Montserrat&quot;,Verdana,sans-serif*/&quot;Simplex_IV25&quot;;
	font-weight: 650;
}</string>
    </property>
    <property name="text">
     <string>см</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Simplex_IV25</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	color: rgb(255, 157, 148);
	border-radius: 20px;
	font-family: /*&quot;Montserrat&quot;,Verdana,sans-serif*/&quot;Simplex_IV25&quot;;
	font-weight: 650;
}</string>
    </property>
    <property name="text">
     <string>ДМ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Simplex_IV25</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 157, 148);
	color: rgb(255, 255, 255);
	border: 1px solid gray;
	border-radius: 20px;
	font-family: /*&quot;Montserrat&quot;,Verdana,sans-serif*/&quot;Simplex_IV25&quot;;
	font-weight: 650;
}</string>
    </property>
    <property name="text">
     <string>мужской</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Simplex_IV25</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	color: rgb(255, 157, 148);
	border-radius: 20px;
	font-family: /*&quot;Montserrat&quot;,Verdana,sans-serif*/&quot;Simplex_IV25&quot;;
	font-weight: 650;
}</string>
    </property>
    <property name="text">
     <string>женский</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>410</y>
      <width>431</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="footer_img.qrc">:/Bez_Imeni-1.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>440</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Simplex_IV25</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(255, 244, 243);
	color: #f00;
	border: 2px solid rgb(255, 82, 85);
    border-radius: 20px;
	font-family: /*&quot;Montserrat&quot;,Verdana,sans-serif*/&quot;Simplex_IV25&quot;;
	font-weight: 650;
}</string>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>350</y>
      <width>39</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Simplex_IV25</family>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font-family: /*&quot;Montserrat&quot;,Verdana,sans-serif*/&quot;Simplex_IV25&quot;;
font-weight: 650;</string>
    </property>
    <property name="text">
     <string>СМ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>350</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font-family: &quot;Montserrat&quot;,Verdana,sans-serif;
font-weight: 600;</string>
    </property>
    <property name="text">
     <string>120</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>430</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Favicon.qrc"/>
  <include location="footer_img.qrc"/>
 </resources>
 <connections/>
</ui>
